docs/modules/user-guide/partials/proc_defining-environment-variables.adoc

```diff
@@ -15,7 +15,7 @@ If the component has multiple containers, environment variables will be provisio
 ----
 schemaVersion: 2.0.0
 metadata:
-  name: MyDevfile
+  name: mydevfile
```

Comment:
`Environment variables are supported by the following component types: container, plugin, kubernetes, openshift.`

Not true. Its only available to container component and exec command(devfile v1 probably did not allow this for a command but devfile v2 does)